README.md

```diff
@@ -1 +1,40 @@
-# New-Portfolio
\ No newline at end of file
+Project Name: This Project is called New-Portfolio
+Project Description: This is a project is intended to showcase my Portfolio as a competent Web developer, to advertise my work and to reach potential employers.
+
+Built With
+Html 
+Css
+Figma
+Live Demo is not available yet.
+
+Getting Started
+This project can be accessed using any web broser.
+
+To get a local copy up and running follow these simple steps.
+1) You can fork this project and use as needed.
+2) You can clone this project using the HTTPS url of this repository.
+
+Prerequisites
+A basic understanding of Html, Css and version control.
+
+Authors
+👤 
+
+GitHub: @uchexm
+Twitter: @nwakauc
+LinkedIn: https://www.linkedin.com/in/uchechi-nwaka-79a21a75/
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
```

Comment:
As suggested by the previous reviewer, I recommend using markdown for your readme to make it more presentable
The previous reviewer has a link you can use to learn this:
This is your current readme look:
<img width="1098" alt="Screenshot 2022-10-27 at 11 31 39" src="https://user-images.githubusercontent.com/48546992/198233914-598f2732-8aa1-4904-a5c0-0a7ff6b5e304.png">
This is what it should look like:
<img width="1213" alt="Screenshot 2022-10-27 at 11 32 03" src="https://user-images.githubusercontent.com/48546992/198234100-1bf45c1d-5ad7-4923-b592-19c478374589.png">
Notice how the content is well arranged. That is as a result of markdown. To make it match the correct format, you can copy the templates markdown by clicking on the copy button shown below:
<img width="1222" alt="Screenshot 2022-10-27 at 11 36 36" src="https://user-images.githubusercontent.com/48546992/198234639-2e478604-63ef-49f5-a92d-3c022e2cfc54.png">
Then edit it to match the information you want displayed.